bulb.py

```diff
@@ -0,0 +1,59 @@
+
+'''Class that contains the set of rules around sensors'''
+class Bulb(object):
+
+    def __init__(self,  name, lightSensor, presenceSensors, homeSensor, wastage_function):
+        self.name = name
+        self.light_sensor = lightSensor
+        self.presence_sensors = presenceSensors
+        self.home_sensor = homeSensor
+        self.wastage_function = wastage_function
+
+
+    def isWasted(self, *args):
+        return self.wastage_function(self,*args)
+
+
+# Actual function implementation for the given lights
+
+def kitchenLampWasted(self):
```

Comment:
this function references `self` but it looks like it's not inside the `class Bulb`? The indentation says it isn't inside the class. Is that desired?